CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 6.3.0 (Apr 18, 2022)
+
+IMPROVEMENTS:
+
+* resource/artifactory_permission_targets: Add deprecation message [GH-413]
+* Removed dependency on `jfrog-client-go` package [GH-413]
+
+NOTES:
+
+* Resource `artifactory_permission_targets` is deprecated and will be removed in next major release. Resource `artifactory_permission_target` (singular) has identical schema which will allow straightforward migration.
```

Comment:
This deprecation notice is obviously not right. Perhaps changing it to -in a future major release"

Reply:
Why is this not right? Next major release will be v7 which we haven't even planned yet.